.github/workflows/ci.yml

```diff
@@ -23,6 +23,27 @@ jobs:
       - uses: olafurpg/setup-scala@v7
         with:
           java-version: "adopt@1.${{ matrix.java }}"
+      - uses: actions/cache@v1
```

Comment:
How about using `COURSIER_CACHE=$HOME/.cache/coursier` so we can use the same directory for all operating systems?

Reply:
I was thinking about it, will change before going out of draft.